### DISCUSSION TITLE
Difference between Supervised and Manual Topic Modeling

### DISCUSSION THREAD

**Discussion by vantubbe** [MaartenGr/BERTopic#1704] [Q&A]
I'm a bit confused on when you would apply one vs the other. In the guide they appear very similar, and supervised is called out as being more of a classification approach. Is manual like this as well? Any chance someone could give a short comparison of the two? Thank you in advance!

**Comment by MaartenGr** [comment]
Manual BERTopic does not fit an underlying model except for the c-TF-IDF representations to create the topic representations. Supervised uses an underlying classification algorithm to learn the relationship between input embeddings and the targets whilst also creating the topic representations. Supervised is an extension of manual. 

I would highly advise going through the documentation [here](https://maartengr.github.io/BERTopic/getting_started/manual/manual.html) and [here](https://maartengr.github.io/BERTopic/getting_started/supervised/supervised.html) and do the examples to get a more intuitive feeling of this.
